### PR TITLE
Fix for delegable API resources not being filtered to the user selected language

### PR DESCRIPTION
**Pull request by jonkjetiloye**
## Description
- Makes user authentication required for the endpoint in order to get users selected language from profile
- Removes having to set {Party} in path as support for filtered result based on reportee access is not supported

## Related Issue(s)
- #303

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/ResourceController.cs [hunk 1]
```diff
@@ -43,19 +43,17 @@ public class ResourceController : ControllerBase
...
-        [Route("accessmanagement/api/v1/{party}/resources/maskinportenschema")]
-        public async Task<ActionResult<List<ServiceResourceFE>>> Get([FromRoute] int party)
+        [Authorize]
+        [Route("accessmanagement/api/v1/resources/maskinportenschema")]
+        public async Task<ActionResult<List<ServiceResourceFE>>> Get()
         {
-            string languageCode = "nb";
+            int userId = AuthenticationHelper.GetUserId(_httpContextAccessor.HttpContext);
+            UserProfile userProfile = await _profileService.GetUserProfile(userId);
+            string languageCode = userProfile?.ProfileSettingPreference?.Language ?? "nb";
```
Comment (acn-dgopa) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/ResourceController.cs:54]: vi har en hjelper metode som gjør det string languageCode = ProfileHelper.GetLanguageCodeForUser(userProfile);

**Review by acn-dgopa — APPROVED** [review]

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/ResourceController.cs (hunk 1 above)
Comment (jonkjetiloye) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/ResourceController.cs:54]: Yes, but that only returns correct language code based on the name of the language files for the application (no_nb, no_nn, en) while resource-registry (and other A3 implementations) use the 2-character language codes as returned by the SBL Bridge user profile integration.

Comment (jonkjetiloye) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/ResourceController.cs:54]: Ahh, sorry my bad. I thought you referred to the GetStandardLanguageCodeForUser.
I can update to use the ProfileHelper.GetLanguageCodeForUser(userProfile);

Comment (acn-dgopa) [reply backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/ResourceController.cs]: yeah :)

**Pull request merged**